### ISSUE TITLE
2.1RC3 Board descriptions, BBC support incomplete

### ISSUE THREAD

**Issue by LexArma** [SimpleMachines/SMF#6345]
#### Description
When creating a board, we have this to advice the admin on how to format the description:
"A short description of your board. HTML is not allowed, you can use the following BBC tags: abbr, anchor, b, center, color, font, hr, i, img, iurl, left, li, list, ltr, pre, right, s, sub, sup, table, td, tr, u, url" 

All these BBC tags do seem to work, on the forum index - But as soon as you enter the board, the description text shown at the topic index spews out BBC tags. 

( Also, the description box seems to accept `<br>`, but tends to lose it if you modify the description later on - since [br] is no longer an option, I do think accepting `<br>` is nothing but a good thing, but ... )

![ForumIndex](https://user-images.githubusercontent.com/39050447/99283878-42478980-283e-11eb-9737-8def94a17569.PNG)
![TopicIndex](https://user-images.githubusercontent.com/39050447/99283879-42e02000-283e-11eb-93a8-da6c1e281998.PNG)

### Steps to reproduce
1. Create a board, adding BBC formatting to the description.
2. See forum index, and compare to topic index inside said board

**Comment by sbulen** [comment]
A dupe of #6307 ...  But it looks like that was closed in error.  Thanks for catching that.

**Comment by LexArma** [comment]
No problem :)

**Comment by live627** [comment]
https://github.com/SimpleMachines/SMF2.1/pull/5796

More specifically,
https://github.com/MissAllSunday/SMF2.1/commit/ffd5d22a589e3db5ec814f749c854148d0e3027f

**Comment by LexArma** [comment]
Maybe a minor nitpick, but looking at the screenshots I made and attached above, I'd also suggest uniforming the layout of the board title and description, so that it's the same on both indexes - using formatting on something that is multilined in one place, and compacted in another makes things "interesting" for the admin.  [/OCD]

**Comment by MissAllSunday** [comment]
Are you using the latest build?

Do you use any cache engine, if so, which one?

**Comment by LexArma** [comment]
RC3 from the SM.org downloads, haven't been keeping up with GH. 
Caching seems to have no effect either way, currently caching is turned off and cache cleared.

**Comment by MissAllSunday** [comment]
Please try the current build. RC3 doesn't contain the fix added in https://github.com/MissAllSunday/SMF2.1/commit/ffd5d22a589e3db5ec814f749c854148d0e3027f

**Comment by LexArma** [comment]
Alright, I'll check that out tomorrow.

**Comment by LexArma** [comment]
Okay, just I made a clean install from albertlast's nightlies ( Last commit 5d327fb ) and the actual issue with rendering the BBC seems to have been fixed as you said. Didn't test much, just made up some boards using b, u and hr. The presented layout could still do with some polish - but that's a different issue. 
![last_commit_5d327fb-ForumIndex](https://user-images.githubusercontent.com/39050447/99901356-ddb58000-2cbe-11eb-93f3-a94c5841d948.PNG)
![last_commit_5d327fb-BoardIndex](https://user-images.githubusercontent.com/39050447/99901360-e0b07080-2cbe-11eb-8aa0-856e21ec295b.PNG)
